scripts/export.js

```diff
@@ -1,115 +1,122 @@
-var Promise = require('bluebird');
-var program = require('commander');
-var pgClient = require('pg').Client;
-var Tar = require('tar-stream');
-var colors = require('colors');
-var fs = Promise.promisifyAll(require('fs'));
-var path = require('path');
-
-var client;
-var projectID;
-var project;
-var destination;
+"use strict";
+
+const Promise = require(`bluebird`);
+const program = require(`commander`);
+const PgClient = require(`pg`).Client;
+const Tar = require(`tar-stream`);
+const colors = require(`colors`);
+const fs = Promise.promisifyAll(require(`fs`));
+const path = require(`path`);
+
+let client;
```

Comment:
if you use `let`, assign it immediately. In this case, as top level variables without a block scope, these should just be `var`